Remotion/ObjectBinding/Web/UI/Controls/BocColumnDefinition.cs

```diff
@@ -100,6 +101,18 @@ public virtual IconInfo ColumnTitleIconDisplayValue
       get { return ColumnTitleIcon; }
     }
 
+    /// <summary> Gets or sets a flag that determines whether to show this column. </summary>
+    [PersistenceMode(PersistenceMode.Attribute)]
+    [Category("Behavior")]
+    [Description("A flag determining whether to show this columns.")]
```

Comment:
```suggestion
    [Description("A flag determining whether to show this column.")]
```

Reply:
I love that I inadvertently copied it from the one single instance with the typo ;)